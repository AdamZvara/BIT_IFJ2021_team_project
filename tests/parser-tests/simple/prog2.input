require "ifj21"

global foo : function(integer)

function bar(par : integer)
	local 
	a : integer = 5
	if a > 2 
		then
	foo(a)
	write(par,"\n")
	else
	end
end

function foo(par : integer)
	if par < 100 then
		write(par," ", "funkce foo\n")
	else
	end end

bar(2)
