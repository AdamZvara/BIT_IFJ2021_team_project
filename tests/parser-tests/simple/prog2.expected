5 funkce foo
2
